docs/pbc/all/payment-service-provider/202410.0/base-shop/payment-method-strategies.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Payment method strategies
+description: This doc describes the different payment method strategies available in Spryker Commerce OS.
```

Comment:
Is this page for ACP app developers?

the audience is not clear.

Reply:
It's for everyone, its a general description.